### PR TITLE
[FIX] Range Slider Value label has bug in RTL

### LINKED ISSUE #8437: [BUG] [UI] Range Slider Value label has bug in RTL
Hi guys,
See below image:

### PR DESCRIPTION
**Pull request by cyclops24**
@RocketChat/core 

Closes #8437 

Before:

After:

### CODE REVIEW

File: packages/rocketchat-theme/client/imports/components/slider.css [hunk 1]
```diff
@@ -17,56 +16,57 @@
...
 	text-align: center;
-
 	border-radius: 3px;
-
 	line-height: 20px;
 }
 
+.rtl .range-slider-value {
+  margin-right: 8px;
+  margin-left: unset;
```
Comment (rodrigok) [inline packages/rocketchat-theme/client/imports/components/slider.css]: Please use the same indentation style

File: packages/rocketchat-theme/client/imports/components/slider.css [hunk 2]
```diff
@@ -17,56 +16,57 @@
...
 	appearance: none;
 }
 
 .range-slider-range::-moz-range-thumb {
 	width: 20px;
 	height: 20px;
-
 	cursor: pointer;
 	transition: background 0.15s ease-in-out;
-
```
Comment (rodrigok) [inline packages/rocketchat-theme/client/imports/components/slider.css]: Do not introduce unnecessary changes

File: packages/rocketchat-theme/client/imports/components/slider.css (hunk 1 above)
Comment (cyclops24) [reply packages/rocketchat-theme/client/imports/components/slider.css]: @rodrigok I done it.

File: packages/rocketchat-theme/client/imports/components/slider.css (hunk 2 above)
Comment (cyclops24) [reply packages/rocketchat-theme/client/imports/components/slider.css]: @rodrigok I done it.

**Comment by cyclops24** [comment]
@rodrigok Can you merge it?

**/CC:** @engelgabriel

**Comment by rodrigok** [comment]
@karlprieb Can you review?
